velox/exec/Task.cpp

```diff
@@ -935,6 +936,32 @@ void Task::addHashJoinBridgesLocked(
   }
 }
 
+void Task::addCustomJoinBridgesLocked(
+    uint32_t splitGroupId,
+    const std::vector<std::shared_ptr<const core::PlanNode>>& planNodes) {
+  auto& translators = Operator::translators();
+  if (0 == translators.size()) {
+    return;
+  }
+
+  auto& splitGroupState = splitGroupStates_[splitGroupId];
+  for (const auto& planNode : planNodes) {
+    for (const auto& translator : translators) {
+      if (auto joinBridge = translator->translate(planNode)) {
+        splitGroupState.bridges.emplace(planNode->id(), joinBridge);
+        return;
+      }
+    }
+  }
+}
+
+// Returns a Customized Join Bridge for 'planNodeId'.
```

Comment:
Method comments should go into the header file.

Reply:
Fixed